### ISSUE TITLE
A32N engine accelerating and decelerating during take off

### ISSUE THREAD

**Issue by Sami244110** [flybywiresim/a32nx#5377]
### Aircraft Version

Stable

### Build info

```json
{
  "built": "2021-05-05T23:03:06+00:00",
  "ref": "v0.6.1",
  "sha": "4a1cccc90ab3997109e394402ab9b41a80fbf325",
  "actor": "Benjozork",
  "event_name": "manual"
}
```

### Describe the bug

A32N engine accelerating and decelerating during take off

### Expected behavior

engine should be stable and constant during take off

### Steps to reproduce

During take off the engines start behaving in a wrong and wearied way

**Comment by tracernz** [comment]
What happens on the PFD and FMA? Is your thrust lever deadband too small on the FLX or TOGA detent?

**Comment by Watsi01** [comment]
Could all be related to current MSFS live weather bug (temps too high).

**Comment by Silvaronn** [comment]
did you correctly set the flex temperature?

**Comment by MisterChocker** [comment]
This issue should be fixed in current master (development version).

**Comment by Sami244110** [comment]
Thank you very much guys for your feedbacks that I really appreciate.

It could be from a wrong calculation of flex temperature or just a bug
The PFD and FMA look normal.

Anyway it doesn't happen often most of the time it works fine some forums
talked about this issue and said it could be from high airport elevation,
however I don't think.

Thank you very much everyone.

**Comment by 2hwk** [comment]
Closing as this issue is already solved in dev/will be solved on release 0.7.0
